docs/setup_selfbuild.md

```diff
@@ -1,17 +1,20 @@
 These instructions are to build and run mjolnir without using [Docker](./setup_docker.md).
-You need to have installed `yarn` 1.x and Node 16.
+To build mjolnir, you have to have installed [Node >16](https://nodejs.org/en/download/), [npm](https://docs.npmjs.com/cli/v7/configuring-npm/install) and [yarn >1.x](https://classic.yarnpkg.com/en/docs/install).
+
+Get the latest release-version https://github.com/matrix-org/mjolnir/releases/latest/ - for example `v1.6.1`.
```

Comment:
It doesn't seem right that we should mention a specific version, it will be out of date.

Reply:
I tried to find something like --latest-release but i could not find anything unfortunately. I just wanted to get somethig to use later when cloning. But i would still leave it that way.

Reply:
We should probably say something like (Note: This example will be out of date) if we leave it.

Reply:
Do you really think it needs to be there? I don't like it that much, but i will try to restructure the text.